datafusion/core/src/datasource/file_format/arrow.rs

```diff
@@ -105,7 +105,7 @@ impl FileFormat for ArrowFormat {
 const ARROW_MAGIC: [u8; 6] = [b'A', b'R', b'R', b'O', b'W', b'1'];
 const CONTINUATION_MARKER: [u8; 4] = [0xff; 4];
 
-/// Custom implementation of inferring schema. Should eventually be moved upstream to arrow-rs. 
+/// Custom implementation of inferring schema. Should eventually be moved upstream to arrow-rs.
 /// See https://github.com/apache/arrow-rs/issues/5021
```

Comment:
```suggestion
/// See <https://github.com/apache/arrow-rs/issues/5021>
```

Copied from `cargo doc`'s output. First heard of [this lint](https://github.com/rust-lang/rust/issues/77501) to me 🫣